kratos/spatial_containers/spatial_search_result.h

```diff
@@ -170,6 +170,27 @@ class SpatialSearchResult
     ///@name Private Operations
     ///@{
 
+    ///@}
+    ///@name Serialization
+    ///@{
+
+    friend class Serializer;
+
+    void save(Serializer& rSerializer) const
+    {
+        rSerializer.save("Object", mpObject);
+        rSerializer.save("Distance", mDistance);
+        rSerializer.save("Is Object Found", mIsObjectFound);
```

Comment:
Just a recommendation, but better not use spaces in the name.

Reply:
Okay